src/views/Compose.vue

```diff
@@ -371,6 +374,11 @@ export default {
             desc: `The percentage of the observation that must be completed to mark the request as complete
             and avert rescheduling. The percentage should be set to the lowest value for which the amount
             of data is acceptable to meet the science goal of the request.`
+          },
+          mosaic: {
+            invalid_parameters_feedback: `The limit to the number of mosaic pointings that can be generated is ${mosaicMaxNumPointings}.
```

Comment:
"that can be generated is ..." -> "that can be generated in the frontend is ..."